src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,259 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.File;
+
+public class Duke {
```

Comment:
There are little or no [javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments) in your files. It would be good to include them to summarize what complex methods are doing.